pytest/tests/loadtest/locust/common/sharding.py

```diff
@@ -346,16 +352,24 @@ def test_random_account(self):
         for (lower, upper) in test_cases:
             # sanity check the test case itself
             if lower is not None:
-                assert account_regex.fullmatch(lower) is not None
+                assert TestRandomAccount.ACCOUNT_REGEX.fullmatch(
+                    lower) is not None
             if upper is not None:
-                assert account_regex.fullmatch(upper) is not None
+                assert TestRandomAccount.ACCOUNT_REGEX.fullmatch(
+                    upper) is not None
 
             for _ in range(10):
                 account_id = random_account_between('foo.near', '_ft', lower,
                                                     upper)
-                assert account_regex.fullmatch(account_id) is not None, (
+                assert TestRandomAccount.ACCOUNT_REGEX.fullmatch(account_id) is not None, (
                     account_id, lower, upper)
                 if lower is not None:
                     assert account_id >= lower, (account_id, lower, upper)
                 if upper is not None:
                     assert account_id < upper, (account_id, lower, upper)
+
+    def test_random_account_for_long_base_name(self):
```

Comment:
also, not a big deal, but you could consider just adding this account ID to the `test_cases` in the existing test, since the thing we're doing there is basically the same as this

Reply:
done.